### PR TITLE
yarn run-ios now runs pod install

### PR DESCRIPTION
**Pull request by tstirrat**
tested locallly, worked fine

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "yarn && react-native start",
     "run-android": "react-native run-android",
-    "run-ios": "react-native run-ios",
+    "run-ios": "cd ios && pod install && cd .. && react-native run-ios",
```
Comment (kenpugsley) [inline package.json:8]: Wouldn't it be slightly better to do the same as the **install:pod**?  
e.g. `cd ios && bundle install && bundle exec pod install && cd .. && react-native run-ios`

Comment (tstirrat) [reply package.json]: Done, switched to using the other script

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "yarn && react-native start",
     "run-android": "react-native run-android",
-    "run-ios": "cd ios && pod install && cd .. && react-native run-ios",
+    "run-ios": "yarn install:pod && react-native run-ios",
```
Comment (kenpugsley) [inline package.json:8]: are we assuming everyone will use yarn?

Comment (kenpugsley) [reply package.json:8]: sorry ... I meant that to be a comment, not request changes.  If realistically everyone must use yarn, then I think this is OK.  I'm just not sure.

Comment (tstirrat) [reply package.json:8]: Oh I see. yes we enforce yarn for this repo (over npm). so they wont get far without it

**Pull request merged**